libs/core/src/components/pds-popover/docs/pds-popover.mdx

```diff
@@ -0,0 +1,157 @@
+import { DocArgsTable, DocCanvas } from '@pine-ds/doc-components';
+import { components } from '../../../../dist/docs.json';
+
+# Popover
+Popovers display contextual information when users click or initiates a keyboard event on an element. They typically contain more information than tooltips and can include interactive elements.
+
+## Guidelines
+### Usage
+#### When to use
+- Use popovers for complex information that might overwhelm the main content
+- When content needs to include interactive elements
+- For non-essential information that requires user interaction to display
+
+#### When not to use
+- Don't use popovers for critical information that must be readily available
+- Avoid using for simple text-only tooltips
+- Don't nest popovers within other popovers
+
+## Properties
+<DocArgsTable componentName="pds-popover" docSource={components} />
+
+### Behavior Types
+#### Auto (Default)
+Closes automatically when clicking outside the popover content. Can also be closed by pressing the `Escape` key.
+
+<DocCanvas client:only
+  mdxSource={{
+    react: `
+      <PdsPopover componentId="auto" text="Auto Popover">
+        <p>Clicks outside will close this popover</p>
+      </PdsPopover>
+    `,
+    webComponent: `
+      <pds-popover component-id="auto" text="Auto Popover">
+        <p>Clicks outside will close this popover</p>
+      </pds-popover>
+    `
+  }}>
+  <pds-popover component-id="auto" text="Auto Popover">
+    <p>Clicks outside will close this popover</p>
+  </pds-popover>
+</DocCanvas>
+
+#### Manual
+Popovers with the `popoverType="manual"` require explicit user interaction to close. This means they won't automatically disappear when clicking outside or pressing Escape.
+
+- add `popoverTargetAction="toggle"` to the popover to allow the user to close it by clicking the trigger button.
+
+{/* To implement this:
+
+1. Create a Closing Function:
+
+    - Define a JavaScript function (e.g., `handleManualClose`) that will be responsible for closing the popover.
+    - Inside this function, you'll typically use `document.getElementById('manual').hide()` to call the `hide()` method on the popover element.
+    - Important: Ensure the id (`manual` in this example) matches the componentId prop of the `PdsPopover` component.
+
+2. Attach Function to onClick:
+
+    - Assign the `handleManualClose` function to the `onClick` event of the button within the popover.
+    - This ensures that when the user clicks the "Close Popover" button, the `handleManualClose` function is executed, and the popover is hidden. */}
+
+export const handleManualClose = () => {
```

Comment:
should this function be referenced in the example below `<pds-popover onClick="handleManualClose" ...>`